playbooks/wazuh-opensearch-production-ready.yml

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
Can we change this file name by `wazuh-indexer-production-ready.yml`?

Reply:
yes

Reply:
This was changed to `wazuh-production-ready.yml`.